.github/workflows/ci-validate-platforms.yml

```diff
@@ -3,224 +3,109 @@ name: Validate Platforms/Browsers
 on:
   push:
     branches:
-    - master
+      - master
+
   pull_request:
     branches:
-    - master
+      - master
+
   schedule:
     - cron: 0 7 * * 3
 
 jobs:
-  linux_cross-platform_cross-browser:
-    runs-on: [ubuntu-latest]
-    env:
-      PLAYWRIGHT_BROWSERS_PATH: 0
+  cross-platform_cross-browser:
+    runs-on: ${{ matrix.os }}
 
-    steps:
-    - uses: actions/checkout@v2
-
-    - name: Get yarn cache directory path
-      id: yarn-cache-dir-path
-      run: echo '::set-output name=dir::$(yarn cache dir)'
-
-    - name: Set up node_modules cache
-      uses: actions/cache@v2
-      id: yarn-cache
-      with:
-        path: |
-          ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          node_modules
-          */*/node_modules
-        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-yarn-
-
-    - name: Install package dependencies
-      run: yarn install --frozen-lockfile --ignore-scripts
-
-    - name: Testing Prettier format
-      run: yarn format:check
-
-    - name: Prepare workspaces
-      run: yarn prepare
-
-    - name: Testing unit tests
-      run: yarn lerna run test --stream
-
-    - name: Install playwright dependencies and browsers
-      run: |
-        yarn playwright install-deps
-        yarn playwright install
-
-    - name: Build Playwright tests
-      run: yarn workspace @microsoft/fast-components build:pw
-
-    - name: Test Playwright (chromium)
-      run: yarn workspace @microsoft/fast-components test-pw
-      continue-on-error: true
-      timeout-minutes: 2
-      env:
-        PLAYWRIGHT_BROWSER: chromium
-
-    - name: Test Playwright (firefox)
-      run: yarn workspace @microsoft/fast-components test-pw
-      continue-on-error: true
-      timeout-minutes: 2
-      env:
-        PLAYWRIGHT_BROWSER: firefox
-
-    - name: Test Playwright (webkit)
-      run: yarn workspace @microsoft/fast-components test-pw
-      continue-on-error: true
-      timeout-minutes: 2
-      env:
-        PLAYWRIGHT_BROWSER: webkit
-
-    - name: Initialize CodeQL
-      if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
-      uses: github/codeql-action/init@v1
-      with:
-        languages: javascript, typescript
-
-    - name: Perform CodeQL Analysis
-      if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
-      uses: github/codeql-action/analyze@v1
-
-  windows_cross-platform_cross-browser:
-    runs-on: [windows-latest]
-    env:
-      PLAYWRIGHT_BROWSERS_PATH: 0
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-11]
```

Comment:
Do you recall why we're running `macos-11` and not `macos-latest` which would be preferred?